execution/executor/tests/storage_integration_test.rs

```diff
@@ -191,7 +195,7 @@ fn test_reconfiguration() {
 
     // rotate the validator's connsensus pubkey to trigger a reconfiguration
     let mut rng = ::rand::rngs::StdRng::from_seed(TEST_SEED);
-    let (_, new_pubkey) = compat::generate_keypair(&mut rng);
+    let new_pubkey = Ed25519PrivateKey::generate(&mut rng).public_key();
```

Comment:
Good point, we cannot use `generate_for_testing` here, as we verify (assert) the seed.

Reply:
actually, this one can be compressed, the test is in another test!